app/import_functions.go

```diff
@@ -923,15 +922,15 @@ func (a *App) ImportAttachment(data *AttachmentImportData, post *model.Post, tea
 
 		if err != nil {
 			fmt.Print(err)
```

Comment:
nit: completely optional improvement here, but we could switch this from using `fmt.Print` to `mlog` for some cleaner error reporting here.